client/src/main/java/se/fortnox/reactivewizard/client/RxClientProvider.java

```diff
@@ -41,6 +44,24 @@ public RxClientProvider(HttpClientConfig config, HealthRecorder healthRecorder)
         return clients.computeIfAbsent(serverInfo, this::buildClient);
     }
 
+    private HttpClient<ByteBuf, ByteBuf> configureSsl(HttpClient<ByteBuf, ByteBuf> client, String host, int port, boolean isUnsafeSecure) {
+        if (isUnsafeSecure) {
+            return client.unsafeSecure();
+        }
+
+        try {
+            SSLEngine sslEngine = sslEngines.computeIfAbsent(host + ":" + port, s -> {
```

Comment:
s?

Reply:
Im expanding the naming to hostPortValue, but it is in effect ignored. I couldn't find any better way while still using the nice computeIfAbsent.